### PR TITLE
Add dependabot and other config files

### PR DESCRIPTION
**Pull request by tsalo**
Closes none.

Changes proposed:

- Add issue templates.
- Add pull request template.
- Add dependabot config file.
- Add release notes config file.

### CODE REVIEW

**Comment by tsalo** [comment]
@mattcieslak I don't think this requires a close review, but I thought I should get your thoughts before merging.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+---
+## Summary
+<!--What is the nature of the bug?-->
+
+## Additional details
+<!--Please fill in the following details-->
+- CuBIDS version:
+- Docker version:
+- Singularity version:
```
Comment (mattcieslak) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]: I vote we pre-emptively remove these

**Review by mattcieslak — APPROVED** [review]
Looks great!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (tsalo) [reply .github/ISSUE_TEMPLATE/bug_report.md:16]: Fair enough. Will do!

**Pull request merged**